packages/tss-client/src/client.ts

```diff
@@ -435,7 +435,11 @@ export class Client {
 
     // remove references
     delete globalThis.tss_clients[this.session];
-
+    this.sockets.map((soc) => {
+      if (soc && soc.connected) {
+        soc.close();
```

Comment:
Could `soc.disconnect()` not be used here instead?

Reply:
its same, both are synonyms as per docs